app/javascript/react/components/AlbumShowContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react'
+import AlbumShow from './AlbumShow'
+
+const AlbumShowContainer = (props) => {
+  const [ album, setAlbum ] = useState({})
+
+  useEffect (() => {
+    let albumId = props.match.params.id
+
+    fetch(`/api/v1/albums/${albumId}`)
+    .then(response => {
+      if (response.ok) {
+        return response.json()
+      } else {
+        const errror = new Error(`${response.status}: ${response.statusText}`);
+        throw(error)
+      }
+    })
+    .then(fetchedAlbum => {
+      setAlbum(fetchedAlbum)
+    })
+    .catch(error => console.error(`Error in fetch: ${error.message}`));
+  }, [])
+
+  return(
+    <div>
+      <h1>This is the album show container page</h1>
+      <AlbumShow
+        id={album.id}
+        name={album.album}
+        art={album.art}
+        artist={album.artist}
+        year={album.year}
+        genre={album.genre}
+      />
```

Comment:
At this point instead of passing down so many props that all stem from `album`, I would just pass down album={album} and destructure it in the AlbumShow component